bin/update

```diff
@@ -12,19 +12,23 @@ FileUtils.chdir APP_ROOT do
   # This script is a way to update your development environment automatically.
   # Add necessary update steps to this file.
 
```

Comment:
wots this? I don't think I've ever run this!

Reply:
I updated it because there was a yarn updated. A script that you can run after pulling changes to get up to date! Been in Rails a while, I think.

Reply:
I should have done it in another branch really.